pymc3/distributions/distribution.py

```diff
@@ -487,7 +487,7 @@ def _draw_value(param, point=None, givens=None, size=None):
 
                 dist_tmp.shape = distshape
                 try:
-                    dist_tmp.random(point=point, size=size)
+                    return dist_tmp.random(point=point, size=size)
```

Comment:
was this a straightup bug here?

Reply:
No bug. But if the try succeeded, then exactly the same code was run again and returned